cmd/crictl/main.go

```diff
@@ -273,6 +275,7 @@ func main() {
 			} else {
 				Debug = config.Debug
 			}
+			PullImageOnCreate = config.PullImageOnCreate
```

Comment:
curious, why not to add this flag to the command line flag overrides as well?

Reply:
Great question!

The current --no-pull option is an override of a default behavior

This new config option is a way to set the default behavior.

------------

I'll add a --with-pull -p option for create.

-------------

I also think it would be fair to have a default config for create.. and a different default config for the run command... The code in the run command currently runs the pod, then calls the create command then the start command.  Thus the run command being more of a helper command lends the user to have different expectations.  I'll add a default config for pull with run.

Thx for the feedback!